### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by priettt**

### CODE REVIEW

File: FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/view/ActivityView.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.pps.globant.fittracker.mvp.view;
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.Context;
+import android.support.annotation.Nullable;
+import java.lang.ref.WeakReference;
+
+//???
+//Deprecated
```
Comment (capraber) [inline FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/view/ActivityView.java]: Please remove this comments lines

File: FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.pps.globant.fittracker.mvp.presenter;
+
+public class LoginPresenter {
```
Comment (capraber) [inline FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java]: Please instantiate this presenter with s simple activity with a place holder text

**Review by vsmartinez — CHANGES_REQUESTED** [review]
Add gradle/ folder to .gitIgnore

Rule: Do not upload Generated files.

File: FitTracker/app/src/main/java/com/pps/globant/fittracker/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import com.pps.globant.fittracker.mvp.presenter.LoginPresenter;
+import com.pps.globant.fittracker.mvp.view.LoginView;
+import com.pps.globant.fittracker.utils.BusProvider;
+
+import butterknife.ButterKnife;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    LoginPresenter presenter;
```
Comment (nacho-globant) [inline FitTracker/app/src/main/java/com/pps/globant/fittracker/MainActivity.java]: Mark as final

File: FitTracker/app/src/main/java/com/pps/globant/fittracker/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        ButterKnife.bind(this);
+        if (presenter == null) {
```
Comment (nacho-globant) [inline FitTracker/app/src/main/java/com/pps/globant/fittracker/MainActivity.java]: you dont need this condition in this place

File: FitTracker/app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/hello_label"
+        android:layout_width="wrap_content"
+        android:textSize="40sp"
```
Comment (nacho-globant) [inline FitTracker/app/src/main/res/layout/activity_main.xml]: create an item into dimens.xml for use that value

**Review by nacho-globant — COMMENTED** [review]
some comments

**Comment by capraber** [comment]
+1

**Comment by nacho-globant** [comment]
+1

**Pull request merged**